src/components/expandable/ExpandableWithSelection.tsx

```diff
@@ -0,0 +1,173 @@
+import { View, Text, TextInput, Pressable, Image } from "react-native";
+import * as React from "react";
+import Icon from "react-native-vector-icons/MaterialIcons";
+import Pen from "react-native-vector-icons/Ionicons";
+import { List } from "react-native-paper";
+import {
+  responsiveHeight,
+  responsiveWidth,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import { ExpandableWithSelectionInterface } from "../../interface/expandable/ExpandableWithSelectionInterface"
+import {Colors} from '../../theme/ColorsConstant'
+
+export const ExpandableWithSelection: React.FC<
+  ExpandableWithSelectionInterface
+> = ({
+  icon = '',
+  iconInfo = '',
+  expandableParagraph = '',
+  userRole = '',
+  successColor = Colors.backgroundColor,
```

Comment:
change color names - with proper name and update in interface as well

Reply:
done